gulp-core-build-sass/src/SassTask.ts

```diff
@@ -26,6 +26,11 @@ export interface ISassTaskConfig {
    */
   useCSSModules?: boolean;
   /**
+   * If true, we will set the CSS property naming warning to verbose message while the module is generating
```

Comment:
Can you change the variable to something like: `warnOnCssTypeSafety` and have it default to `true`? It would fit in a bit better with the variable `warnOnNonCSSModules` below. A description could be: "If true, the type-safety warning around using the "-" character in CSS classes will be logged as a warning. If false, it will be logged only during verbose mode. Default: `true`"